### ISSUE TITLE
Dynamic @entangle parameter throwing error

### ISSUE THREAD

**Issue by jjjrmy** [livewire/livewire#1490]
```javascript
x-data="{
    address: @entangle('a.{{ $input->id }}').defer,
}"
```
Throws Error: `syntax error, unexpected '}', expecting ')'`

```javascript
x-data="{
    address: @entangle('a.<?= $input->id; ?>').defer,
}"
```
Throws Error: `syntax error, unexpected 'instanceof' (T_INSTANCEOF)`

**Comment by imliam** [comment]
This particular error is because it's just not valid syntax - `@entangle` is a Blade directive so needs regular PHP in its parenthesis, eg:

```
@entangle('a.' . $input->id)
```

That said, we discussed this on Discord and found that this syntax didn't work either because of an operator precedence issue, so I fixed that in #1492

**Comment by calebporzio** [comment]
Liam is right. Thanks!
